### PR TITLE
Random maint loot and miscellaneous things

### PR DESCRIPTION
**Pull request by Alberyk**
Things done:
- Created several random objects, that can be used to randomize loot in the maint or other places. Several items, some rare things and other random useful or not objects.
- Added a svd gun with its own snowflakness, to replace the heister's lwap, but, they will get an extra energy gun to compesate.
- Added tracksuits, stunrod, new sprite for the rapier, a new sword flavor and the old cane sword.
- Changed the heister ship armory; two random melee weapons, the dragunov with an extra mag and a random energy gun.
- Changed the chances of ghetto guns going bad.
- Replaced the carbines in the ert area with energy pistols, so, they can make use of the holsters.
- Clumsy people may have trouble with glowsticks now.

### CODE REVIEW

**Comment by Alberyk** [comment]
And this is probably done.

**Comment by Alberyk** [comment]
Done.

**Comment by skull132** [comment]
Map conflict, probably from Fowl.

You two are even now.

**Comment by Alberyk** [comment]
Fixed merge conflicts.

File: code/game/objects/items/weapons/material/swords.dm [hunk 1]
```diff
@@ -70,7 +70,7 @@
 	
 /obj/item/weapon/material/sword/axe
 	name = "battle axe"
-	desc = "A one handed battle axe, still a deadly weapon."
+	desc = "An one handed battle axe, still a deadly weapon."
```
Comment (skull132) [inline code/game/objects/items/weapons/material/swords.dm]: Actually. Hate to be a downer, but it's still "A one handed battle axe." The article is dependent on the noun it's associated with (battle axe), not the adjectives (one handed) that come before it.

File: code/game/objects/items/weapons/material/swords.dm [hunk 2]
```diff
@@ -65,3 +67,15 @@
 	icon_state = "sabre"
 	item_state = "katana"
 	slot_flags = SLOT_BELT
+	
+/obj/item/weapon/material/sword/axe
+	name = "battle axe"
+	desc = "An one handed battle axe, still a deadly weapon."
```
Comment (skull132) [inline code/game/objects/items/weapons/material/swords.dm]: Actually. Hate to be a downer, but it's still "A one handed battle axe." The article is dependent on the noun it's associated with (battle axe), not the adjectives (one handed) that come before it.

File: code/game/objects/random/random.dm [hunk 3]
```diff
@@ -278,3 +276,234 @@
...
+	"/obj/item/clothing/head/pumpkinhead" = 0.6,
+	"/obj/item/clothing/head/kitty" = 0.2,
+	"/obj/item/clothing/head/ushanka" = 0.6,
+	"/obj/item/clothing/head/helmet/augment" = 0.1,
+	"/obj/item/clothing/mask/balaclava" = 1,
+	"/obj/item/clothing/mask/gas" = 1.5,
+	"/obj/item/clothing/mask/gas/cyborg" = 0.7,
+	"/obj/item/clothing/mask/gas/owl_mask" = 0.8,
+	"/obj/item/clothing/mask/gas/syndicate" = 0.4,
+	"/obj/item/clothing/mask/fakemoustache" = 1,
```
Comment (skull132) [inline code/game/objects/random/random.dm]: Doesn't the fake moustache allow you to change your name and stuff? Ergo, is a syndicate item?

File: code/game/objects/random/random.dm [hunk 4]
```diff
@@ -278,3 +276,234 @@
...
+					/obj/item/weapon/material/sword/rapier)
+
+/obj/random/coin
+	name = "Random Coin"
+	desc = "This is a random coin."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "coin"
+	item_to_spawn()
+		return pick(prob(5);/obj/item/weapon/coin/iron,\
+					prob(3);/obj/item/weapon/coin/silver,\
```
Comment (skull132) [inline code/game/objects/random/random.dm]: Still has probabilities attached.

Meaning a probability is generated for every item on this list, every time a random coin is spawned.

File: code/game/objects/items/weapons/stunbaton.dm [hunk 5]
```diff
@@ -203,3 +203,32 @@
...
+		icon_state = "[initial(name)]_active"
+		item_state = "[initial(name)]_active"
+	else if(!bcell)
+		icon_state = "[initial(name)]_nocell"
+		item_state = "[initial(name)]"
+	else
+		icon_state = "[initial(name)]"
+		item_state = "[initial(name)]"
+
+	..()
```
Comment (skull132) [inline code/game/objects/items/weapons/stunbaton.dm]: Was this tested? Referring to parent here may cause the baton's own definition (which apparently doesn't fit) to take over, and modify the icon_state and item_state.

**Comment by Alberyk** [comment]
The lists should be better now, also, fixed the coin one.

**Comment by Arrow768** [comment]
As discussed please take a look at the short / absolute paths in the random.dm

**Comment by skull132** [comment]
Conflicts from the cargo warehouse PR.

**Pull request merged**